types/relay-test-utils/lib/RelayMockPayloadGenerator.d.ts

```diff
@@ -1,27 +1,21 @@
-import {
-    Variables,
-    NormalizationField,
-    NormalizationOperation,
-    NormalizationSelection,
-    NormalizationLinkedField,
-    NormalizationScalarField,
-    OperationDescriptor,
-    GraphQLResponse,
-    NormalizationSplitOperation,
-} from 'relay-runtime';
+import { GraphQLSingularResponse, OperationDescriptor } from 'relay-runtime';
 
-interface MockResolverContext {
-    parentType?: string | undefined;
-    name?: string | undefined;
-    alias?: string | undefined;
-    path?: ReadonlyArray<string> | undefined;
-    args?: { [key: string]: any } | undefined;
+export interface MockResolverContext {
+    parentType?: string | null | undefined;
+    name?: string | null | undefined;
+    alias?: string | null | undefined;
+    path?: ReadonlyArray<string> | null | undefined;
+    args?: { [key: string]: any } | null | undefined;
```

Comment:
According to https://github.com/facebook/relay/blob/8a44673ef671e6e06eae0a4df5fe52da23fde703/packages/relay-test-utils/RelayMockPayloadGenerator.js file, the properties themselves aren't optional:
```
type MockResolverContext = {
  +parentType: ?string,
  +name: ?string,
  +alias: ?string,
  +path: ?$ReadOnlyArray<string>,
  +args: ?{[string]: mixed, ...},
};
```
e.g., `parentType: string | null | undefined`, not `parentType?: `

They're also `readonly` if you wanted to be more exactly matching too